<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QLabel" name="fileLabel">
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Файл профиля: не выбран</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Директория генерации файла &quot;MBedit.c&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolBtnGenPath">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditFieGenDir"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QPushButton" name="startGenBtn">
      <property name="text">
       <string>Генерировать MBedit.c</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    border-radius: 7px;
}

QRadioButton::indicator:unchecked {
    background-color: #808080;
    border: 2px solid #808080;
}

QRadioButton::indicator:checked {
    background-color: white;
    border: 3px solid #808080;
}</string>
      </property>
      <property name="title">
       <string>Формат чисел</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>IQmath</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Float</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabProfile">
       <attribute name="title">
        <string>Карта регистров</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabProfileSettings">
       <attribute name="title">
        <string>Базовые величины</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableWidgetBaseValues"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open"/>
    <addaction name="save"/>
    <addaction name="saveAs"/>
    <addaction name="exportToExcel"/>
    <addaction name="importFromExcel"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="exportToExcel">
   <property name="text">
    <string>Экспорт в Excel</string>
   </property>
  </action>
  <action name="importFromExcel">
   <property name="text">
    <string>Импорт из Excel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
